cypress/e2e/tables-table.cy.js

```diff
@@ -23,13 +23,35 @@ describe('Manage a table', () => {
 		cy.get('.tile').contains('ToDo').click({ force: true })
 		cy.get('.modal__content').should('be.visible')
 		cy.get('.modal__content input[type="text"]').clear().type('to do list')
-		cy.contains('button', 'Create table').click()
+		cy.contains('button', 'Create table').scrollIntoView().click()
 
 		cy.contains('button', 'Create row').should('be.visible')
 		cy.contains('h1', 'to do list').should('be.visible')
 		cy.contains('table th', 'Task').should('exist')
 	})
 
+	it('Create with import', () => {
+		cy.uploadFile('test-import.csv', 'text/csv')
+		cy.contains('.app-menu-entry--label', 'Tables').click()
+		cy.contains('button', 'Create new table').click()
+		cy.get('.tile').contains('Import').click({ force: true })
+		cy.get('.modal__content').should('be.visible')
+		cy.get('.modal__content input[type="text"]').clear().type('import list')
+		cy.contains('button', 'Create table').scrollIntoView().click()
+		cy.contains('h2', 'Import').should('be.visible')
+
+		cy.get('.modal__content button').contains('Select a file').click()
```

Comment:
Failing cypress test indicates that this is not `Select a file` but `Select from Files`. I think it actually should be `Select from files` as we only capitalize the first word.

![image](https://github.com/nextcloud/tables/assets/97337118/ee077d18-232a-4c9a-9c8f-a9fb7ec8c8b6)